lib/documents.js

```diff
@@ -2473,7 +2474,22 @@ function writeDocs(jobState, val, batchdef, writeBatchArray, writerId){
       });
 }
 
-
+/**
+ * The queryAll function reads one or more document uris from database using a passThrough stream and optional options.
+ * The optional options include onCompletion, batchSize, queryBatchMultiple and consistentSnapshot.
+ * @method documents#queryAll
+ * @since 2.9.0
+ * @param {ctsQuery} query - A query built by the ctsQueryBuilder.
+ * @param {object} [options] - Configures the query operation.
+ * @param {function(summary)} [onCompletion] - Provides a summary of the results.
```

Comment:
I'd suggest phrasing that starts with the term "callback" as in 

    A callback that receives a summary of the results.

I'd suggest the same change for writeAll(), as in onBatchSuccess

   A callback that can notify other systems about batches written successfully.